docs/ALPHA.md

```diff
@@ -0,0 +1,13 @@
+# ALPHA
+
+```bash
+TMP=$(az account show)
```

Comment:
This is just a convenience to get your subscription id and tenant id from an az CLI that's already logged in.